validate.py

```diff
@@ -94,9 +93,33 @@ def determine_if_mergeable(pr):
 def determine_if_winner():
   print_points()
 
+  # Pick a winner at random with a single random number.  We divide the number
```

Comment:
#97 changes leaked in.

Reply:
I almost want to throw caution to the wind; this isn't work, this is a *game!* But yeah, we probably should :P

Reply:
No problem!

(This isn't really a caution-to-the-wind sort of game...)